test/Utilities/copy.jl

```diff
@@ -464,10 +464,10 @@ function test_create_variables_using_supports_add_constrained_variable()
 
     dest = OrderConstrainedVariablesModel()
     bridged_dest = MOI.Bridges.full_bridge_optimizer(dest, Float64)
-    @test MOIU.sorted_variable_sets_by_cost(bridged_dest, src) == [
-        (MOI.VectorOfVariables, MOI.Zeros),
-        (MOI.VectorOfVariables, MOI.Nonnegatives),
-        (MOI.VectorOfVariables, MOI.Nonpositives),
+    @test MOIU._sorted_variable_sets_by_cost(bridged_dest, src) == Any[
```

Comment:
It's used by https://github.com/jump-dev/Dualization.jl/pull/112

Reply:
I've changed it around a bit. It's now exported with a docstring, but it doesn't contain the function information because you can infer that from the super type of AbstractScalarSet or AbstractVectorSet.